src/notation/qml/MuseScore/NotationScene/internal/EditStyle/BeamsPage.qml

```diff
@@ -76,44 +76,16 @@ StyleDialogPage {
         }
     }
 
-    ItemWithTitle {
-        title: qsTrc("notation", "Beam thickness")
-
-        IncrementalPropertyControl {
-            width: 106
-
-            currentValue: beamsPageModel.beamWidth.value
-
-            minValue: 0
-            maxValue: 99
-            step: 0.01
-            decimals: 2
-            measureUnitsSymbol: qsTrc("global", "sp")
```

Comment:
* we lose "width: 106" and gain "preferredWidth: 80" which seems fine to me.
* we keep "currentValue", since it's passed in, and inPercentage is false by default.
* we keep "minValue", "maxValue", "step", and "decimals" since again, inPercentage is false.
* we keep "measureUnitsSymbol" since it's passed in as "suffix".

Reply:
Ah, I did lose "step" on "broken beam minimum length". Think I should pass it in, or just not use IncrementalPropertyControl?

Reply:
The usage of `BasicStyleSelectorWithSpinboxAndReset` seems fine to me, but you will need to add some new properties to it, in order to be able to pass the `step` value in.